etc/strings/current_env_content.txt

```diff
@@ -0,0 +1,2 @@
+parse_git_file() { git_file=$(git rev-parse --show-toplevel 2>/dev/null)/.kw/env.current; [ -f "$git_file" ] && printf '[%s]' "$(cat "$git_file")"; }
```

Comment:
The string folder is only for strings that will be displayed to users. You could keep this under the src folder.

Additionally, why `parse_git_file`? Is this a copy & paste issue?

Reply:
I forgot to change that name; it's not intuitive at all. I had left this name while testing how to make it show the contents of _env.current_ in the terminal.

Reply:
Wouldn't it be better to save this file in /etc instead of /src?